README.md

```diff
@@ -32,4 +33,25 @@ docker run -d \
     nginx
 ```
 
+Example of webhooks usage.
+Supply valid JSON as array of webhooks, see structure below:
+```
+{
+    "url": "http://localhost:9090/-/reload", 
+    "method": "POST|GET",
+    "success": 200
+}
+```
+
+```
+# run the container
+docker run -d \
+    -v /tmp/git-data:/git \
+    registry/git-sync:tag \
+        --repo=https://github.com/kubernetes/git-sync
+        --branch=master
+        --wait=30
+        --webhooks='[{"url": "http://localhost:9090/-/reload", "method": "POST", "success": 200}]'
```

Comment:
this might be cleaner as:

```
--webhook='[{"url": "http://localhost:9090/-/reload", "method": "POST", "success": 200}]'
--webhook='[{"url": "http://1.2.3.4:567/hooked", "method": "POST", "success": 404}]'
```